src/installer.core.cybercat.ts

```diff
@@ -0,0 +1,57 @@
+import { win32 } from "path";
+import {
+  VortexApi,
+  VortexLogFunc,
+  VortexTestResult,
+  VortexInstallResult,
+  VortexWrappedInstallFunc,
+  VortexWrappedTestSupportedFunc,
+} from "./vortex-wrapper";
+import { FileTree, FILETREE_ROOT } from "./filetree";
+import { CYBERCAT_CORE_BASEDIR } from "./installers.layouts";
+import { showInfoNotification, InfoNotification } from "./ui.notifications";
+import { instructionsForSourceToDestPairs, moveFromTo } from "./installers.shared";
+import { GAME_ID } from "./index.metadata";
+
+const path = win32;
+
+const CYBERCAT_CORE_IDENTIFIERS = [
+  path.normalize("CP2077SaveEditor.exe"),
+  path.normalize("licenses/CyberCAT.Core.LICENSE.txt"),
+  path.normalize("previews/BodyGender/00.jpg"),
+];
+
+export const testForCyberCatCore: VortexWrappedTestSupportedFunc = (
+  api: VortexApi,
+  log: VortexLogFunc,
+  files: string[],
+  _fileTree: FileTree,
+): Promise<VortexTestResult> => {
+  const containsAllNecessaryCyberCatFiles = CYBERCAT_CORE_IDENTIFIERS.every(
+    (cyberCatPath) => files.includes(cyberCatPath),
+  );
+
+  return Promise.resolve({
+    supported: containsAllNecessaryCyberCatFiles,
+    requiredFiles: [],
+  });
+};
+
+export const installCoreCyberCat: VortexWrappedInstallFunc = (
+  api: VortexApi,
+  log: VortexLogFunc,
+  files: string[],
+  _fileTree: FileTree,
+  _destinationPath: string,
+): Promise<VortexInstallResult> => {
+  log("info", "Using CyberCAT installer");
+
+  const topleveltoCyberCat = files.map(moveFromTo(FILETREE_ROOT, CYBERCAT_CORE_BASEDIR));
+
+  const movingInstructions = instructionsForSourceToDestPairs(topleveltoCyberCat);
+
+  api.emitAndAwait("discover-tools", GAME_ID);
+  showInfoNotification(api, InfoNotification.CyberCatRestartRequired);
+
+  return Promise.resolve({ instructions: movingInstructions });
```

Comment:
Is there a chance that the user will restart before the install completes? I don’t know if even their own tooling accounts for a race condition here though… probably not worth trying to weave the notification through the return chain for this.

Reply:
I really wish there was an easy way to make the notification wait until after install

Reply:
Also, see [Vortex #12073](https://github.com/Nexus-Mods/Vortex/issues/12073).  Once that closes, I think we can get rid of the notification or suggest the user manually refreshes *if necessary*

Reply:
That reminds me, is there a better place to put that `emitAndAwait()` so that it processes after the files are installed?

Reply:
I tried to see if that were the case, but I don't think there is, as far as I can tell.

Reply:
I actually think I have this figured out, but testing outside of "it doesn't crash" can't be done because of the above refenced issue.

Insert the following into `main()`: 
```ts
  vortex.once(() => {
    vortex.api.onAsync(`did-deploy`, (profileId) => {
      const state = vortex.api.store.getState();
      const profile = vortexApi.selectors.profileById(state, profileId);

      if (GAME_ID !== profile?.gameId) {
        return Promise.resolve();
      }

      vortex.api.emitAndAwait(`discover-tools`, GAME_ID);
      return Promise.resolve();
    });
  })
```